pages/en/learn/getting-started/nodejs-with-typescript.md

```diff
@@ -117,6 +117,57 @@ TypeScript offers a whole lot of other great mechanisms like interfaces, classes
 
 Some of the other benefits of TypeScript that are worth mentioning are that it can be adopted progressively, it helps making code more readable and understandable and it allows developers to use modern language features while shipping code for older Node.js versions.
 
+## Running TypeScript code in Node.js
+
+Sadly node can do this `node example.ts`. But there are three solutions to this problem 🥳:
+
+### Compiling TypeScript to JavaScript
+
+If you want to run TypeScript code in Node.js, you need to compile it to JavaScript first. You can do this using the TypeScript compiler `tsc` as shown above.
+
+Here a small example:
+
+```bash
+npx tsc example.ts
+node example.js
+```
+
+### Running TypeScript code with ts-node
+
+You can use [ts-node](https://typestrong.org/ts-node/) to run TypeScript code directly in Node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `ts-node` before shipping it.
+
+To use `ts-node`, you need to install it first:
+
+```bash
+npm i -D ts-node
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+npx ts-node example.ts
+```
+
+### Running TypeScript code with nodejs loader
+
+Since node v19.0.0 we had [custom loader](https://nodejs.org/docs/v20.11.1/api/cli.html#--importmodule). So you i'll need to download an loader such as `ts-node` or `tsx` or `nodejs-loaders`
```

Comment:
grammar and evergreen link

```suggestion
Since Node.js v19.0.0, you can use a [custom loader](https://nodejs.org/api/cli.html#--importmodule). Download a loader such as `ts-node` or `tsx` or `nodejs-loaders`
```